### PR TITLE
Release contributorhome

### PR DESCRIPTION
**Pull request by carlyrevier**
# Contributor guide pull request

Thanks for contributing to the [Microsoft Learn documentation contributor guide](https://learn.microsoft.com/contribute/?branch=main). Please read these instructions for processing your pull request (PR). Once you've read this text, delete it and write a brief description of the changes you're proposing.

## Quality control

- [ ] 1. **Successful build with no warnings**: Review the build status to make sure **all checks are green** (Succeeded).

- [ ] 2. **#Sign-off**: Once the PR is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the PR. Signing off means the document is ready for review and can be published at any time.

## Merge and publish

- All PRs to this repository are manually reviewed and merged.
- Once all feedback on the PR is addressed, we'll merge the PR into the main branch.

## Need help?

- See our guidance on [Contributing to the contributor guide](/contribute/?branch=main) for help with adding or updating content in this contributor guide.
- Write to the PR reviewer in the PR comments: `@carlyrevier, @jehchow`

### CODE REVIEW

**Comment by carlyrevier** [comment]
#sign-off

@jehchow for review

**Review by jehchow — COMMENTED** [review]
@carlyrevier Looks AWESOME. Just a few comments and suggestions below!

Also, just to note, you didn't introduce these, but I found these links that go to internal resources that we should fix at some point.

- In get-started-setup-tools, in step 3, I noticed that the "set up an editor" link is going to the internal guide?

- in text-formatting-guidelines, there's a link to "Describing interactions with UI" that seems to be internal-only

File: Contribute/breadcrumb/toc.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
+items:
 - name: Learn
   tocHref: /
   topicHref: /
   items:
-  - name: Documentation contributor guide
-    tocHref: /contribute/
-    topicHref: /contribute/
+  - name: Contribute
```
[the remark this answers is not in the document]
Comment (carlyrevier) [reply Contribute/breadcrumb/toc.yml:6]: The breadcrumb already works that way. :)

Noted -- I'll change the TOC entry.

**Comment by carlyrevier** [comment]
> @carlyrevier Looks AWESOME. Just a few comments and suggestions below!
> 
> Also, just to note, you didn't introduce these, but I found these links that go to internal resources that we should fix at some point.
> 
> * In get-started-setup-tools, in step 3, I noticed that the "set up an editor" link is going to the internal guide?
> * in text-formatting-guidelines, there's a link to "Describing interactions with UI" that seems to be internal-only

Addressed these in a new commit!

**Comment by carlyrevier** [comment]
#sign-off

**Review by jehchow — APPROVED** [review]

**Pull request merged**